databricks/sdk/_widgets/IPyWidgetUtils.py

```diff
@@ -0,0 +1,89 @@
+import typing
+
+from IPython.core.display_functions import display
+from ipywidgets.widgets import (ValueWidget, Widget, widget_box,
+                                widget_selection, widget_string)
+
+from .WidgetUtils import _WidgetUtils
+
+
+class DbUtilsWidget:
+
+    def __init__(self, label: str, valueWidget: ValueWidget) -> None:
+        self.lavelWidget = widget_string.Label(label)
+        self.valueWidget = valueWidget
+        self.box = widget_box.Box([self.lavelWidget, self.valueWidget])
+
+    def display(self):
+        display(self.box)
+
+    def close(self):
+        self.lavelWidget.close()
+        self.valueWidget.close()
+        self.box.close()
+
+    def __del__(self):
+        self.close()
+
+    @property
+    def value(self):
+        value = self.valueWidget.value
+        if type(value) == str or value is None:
+            return value
+        if type(value) == list or type(value) == tuple:
+            return ','.join(value)
+
+        raise ValueError("The returned value has invalid type (" + type(value) + ").")
+
+
+class IPyWidgetUtil(_WidgetUtils):
+
+    def __init__(self) -> None:
+        self.__widgets: typing.Dict[str, DbUtilsWidget] = {}
+
+    def __register(self, name: str, widget: ValueWidget, label: typing.Optional[str] = None):
```

Comment:
```suggestion
    def _register(self, name: str, widget: ValueWidget, label: typing.Optional[str] = None):
```
one underscore is enough :) I don't think a lot of people will be extending it.